resources.qrc

```diff
@@ -15,7 +15,9 @@
         <file>qml/pages/VenueList.qml</file>
         <file>qml/pages/VenueMapPage.qml</file>
         <file>qml/harbour-berlin-vegan.qml</file>
+        <file>translations/harbour-berlin-vegan-de.qm</file>
```

Comment:
Nice! I have actually been confused all the time how and why the translations were already working on Sailfish.
I assume they have some other mechanism in place that deploys the QM files to the RPM/device.
Are these qm files used for QML, at all?